### PR TITLE
KR User model

### PR DESCRIPTION
**Pull request by kjreczek**
I created the first version of the User Model that you can rely on, there are a few things that can probably be extended here but we can do that later.

### CODE REVIEW

File: Api/Src/Models/User.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import * as mongoose from 'mongoose';
+
+export enum UserType {
+    Participant,
+    Mentor, 
+    Admin,
+};
+
+export interface UserModel {
```
Comment (JustynaSmoliga) [inline Api/Src/Models/User.ts:9]: Model looks good ;) When I've created endpoints documentation I also created user model. Our models are similar, but in my model are additional fields like grades and status. You can check this in my pull request. Maybe we should add these fields into your model? We can talk about it during meeting. ;)

Comment (kjreczek) [reply Api/Src/Models/User.ts:15]: Ok, I added a status to user model and changed the names in the model to be the same. I suggest add grades after creating or adding files about grades. In user model we must add grades model, and this is also not done yet.

Comment (JustynaSmoliga) [reply Api/Src/Models/User.ts:16]: Ok, we will add grades to user when model will be complete.

**Review by JustynaSmoliga — APPROVED** [review]

File: Api/Src/Models/User.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+export enum UserType {
+    Candidate,
+    Participant,
+    Mentor, 
+    Admin,
+};
+
+export enum UserStatus {
+    Active, 
+    Deleted,
```
Comment (JustynaSmoliga) [inline Api/Src/Models/User.ts:13]: In endpointsDocumentation task I have talked with Filip about userStatus. We should rename Deleted status to Resigned to clarify what does it mean.

Comment (kjreczek) [reply Api/Src/Models/User.ts]: Done 👍

**Review by fszwedo — APPROVED** [review]
approving not to block the merge as my propositions were not mandatory ;)

**Pull request merged**